tests/test_install.py

```diff
@@ -29,23 +29,20 @@ def test_install_simple(selenium_standalone_micropip):
 
 
 @pytest.mark.parametrize("base_url", ["'{base_url}'", "'.'"])
-def test_install_custom_url(selenium_standalone_micropip, base_url):
+def test_install_custom_url(selenium_standalone_micropip, base_url, test_wheel_catalog):
     selenium = selenium_standalone_micropip
+    snowball_wheel = test_wheel_catalog.get("snowballstemmer")
+    url = snowball_wheel.url
 
-    with spawn_web_server(TEST_WHEEL_DIR) as server:
-        server_hostname, server_port, _ = server
-        base_url = f"http://{server_hostname}:{server_port}/"
-        url = base_url + SNOWBALL_WHEEL
-
-        selenium.run_js(
-            f"""
-            await pyodide.runPythonAsync(`
-                import micropip
-                await micropip.install('{url}')
-                import snowballstemmer
-            `);
-            """
-        )
+    selenium.run_js(
+        f"""
+        await pyodide.runPythonAsync(`
+            import micropip
+            await micropip.install('{url}')
+            import snowballstemmer
+        `);
+        """
+    )
```

Comment:
Maybe while we're att it we should transform this one to use `@run_in_pyodide`:

```suggestion
    @run_in_pyodide
    def install_from_url(url):
        import micropip
        await micropip.install(url)
        import snowballstemmer

    install_from_url(url)
```